src/Storage/Storage.Management/Blob/DisableAzureStorageBlobDeleteRetentionPolicy.cs

```diff
@@ -0,0 +1,122 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.Storage
+{
+    using Microsoft.Azure.Commands.Management.Storage.Models;
+    using Microsoft.Azure.Management.Storage;
+    using Microsoft.Azure.Management.Storage.Models;
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+    /// <summary>
+    /// Modify Azure Storage service properties
+    /// </summary>
+    [Cmdlet("Disable", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + StorageBlobDeleteRetentionPolicy, SupportsShouldProcess = true, DefaultParameterSetName = AccountNameParameterSet), OutputType(typeof(PSDeleteRetentionPolicy))]
+    public class DisableAzStorageBlobDeleteRetentionPolicyCommand : StorageBlobBaseCmdlet
+    {
+
+        /// <summary>
+        /// AccountName Parameter Set
+        /// </summary>
+        private const string AccountNameParameterSet = "AccountName";
+
+        /// <summary>
+        /// Account object parameter set 
+        /// </summary>
+        private const string AccountObjectParameterSet = "AccountObject";
+
+        /// <summary>
+        /// BlobServiceProperties ResourceId  parameter set 
+        /// </summary>
+        private const string PropertiesResourceIdParameterSet = "BlobServicePropertiesResourceId";
+
+        [Parameter(
+          Position = 0,
+          Mandatory = true,
+          HelpMessage = "Resource Group Name.",
+         ParameterSetName = AccountNameParameterSet)]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            HelpMessage = "Storage Account Name.",
+           ParameterSetName = AccountNameParameterSet)]
+        [ResourceNameCompleter("Microsoft.Storage/storageAccounts", nameof(ResourceGroupName))]
+        [Alias(AccountNameAlias)]
```

Comment:
Is this alias 'Name'?

Reply:
the alias is "AccountName"
`protected const string AccountNameAlias = "AccountName";`

Reply:
Fixed by add alias "Name".
See commit https://github.com/Azure/azure-powershell/pull/8874/commits/8749ddd484c4ae299796c4019ee0843c90249192